config/routes.rb

```diff
@@ -5,9 +5,7 @@
   resources :reports
   resources :pipeline_outputs
   devise_for :users
-  resources :samples do
-    get :insert, on: :collection
-  end
```

Comment:
and similarly i don't know what this does

Reply:
insert no longer exists